test/testlib/nuodb_database_utilities.go

```diff
@@ -143,4 +143,35 @@ func StartDatabase(t *testing.T, namespace string, adminPod string, options *hel
 func SetDeploymentUpgradeStrategyToRecreate(t *testing.T, namespaceName string, deploymentName string) {
 	kubectlOptions := k8s.NewKubectlOptions("", "", namespaceName)
 	k8s.RunKubectl(t, kubectlOptions, "patch", "deployment", deploymentName, "-p", UPGRADE_STRATEGY)
-}
\ No newline at end of file
+}
+
+func PopulateDBWithQuickstart(t *testing.T, namespaceName string, adminPod string, dbName string) {
```

Comment:
just pointing out that these take `forever` to finish... I wish we had something that inserts some data and take less time... Nothing you can do here.

Reply:
Actually it's not that long. Locally it takes ~10sec.